modules/test/playwright/tests/commerce/commerceSearchSuggestions.spec.ts

```diff
@@ -36,7 +36,9 @@ test('LPD-18809 search suggestions should filter by product visibility with the
 
 	// setup
 
-	const site = await apiHelpers.headlessSite.createSite(getRandomString());
+	const site = await apiHelpers.headlessSite.createSite({
+		name: getRandomString(),
+	});
```

Comment:
Already merged?

Reply:
I hope so. I'll wait for git to complain :smile:

Reply:
yes' it's merged this way. I suffered it several times